src/module.c

```diff
@@ -13295,7 +13295,7 @@ int RM_GetKeyspaceNotificationFlagsAll(void) {
  * Example for 6.0.7 the return value will be 0x00060007.
  */
 int RM_GetServerVersion(void) {
-    return SERVER_VERSION_NUM;
+    return VALKEY_VERSION_NUM;
 }
```

Comment:
Not the scope of this PR, but I wonder whether we should return the Redis compat version here instead.